modules/ROOT/pages/guided-setup.adoc

```diff
@@ -20,7 +20,7 @@ To connect your API Community Manager instance to your Anypoint organization:
 
 . Select your organization's Anypoint management plane.
 +
-This may be the US plane, the EU plane, or another plane.
+This may be the US plane, the EU plane, the MuleSoft Government Cloud plane, or another plane.
```

Comment:
The only "plane"s in our term presentation list are "control plane" and "runtime plane."
The only mentions of "EU" or "US" are as types of "cloud."

Please verify this wording; again, I am relying on our source of truth at https://docs.google.com/spreadsheets/d/1f6v9cjx84ZAHGeZs36A0clZoQc_ggAe6m6e-NQ9BUuk/edit#gid=71218867.

GLOBALLY search for and fix these.

Reply:
Should be US control plane and EU control plane. Fixed.